docs/src/main/asciidoc/cdi-integration.adoc

```diff
@@ -190,6 +190,20 @@ AnnotationsTransformerBuildItem transform() {
 
 NOTE: Keep in mind that annotation transformers must be produced _before_ the bean discovery starts.
 
+You can also use a convenient builder-like API to create transformer instances.
+The example above can be rewritten like:
+
+Builder Example
+[source,java]
+----
+@BuildStep
+AnnotationsTransformerBuildItem transform() {
+   return new AnnotationsTransformerBuildItem(AnnotationsTransformer.appliedToClass()
+        .whenClass(c -> c.name().toString().equals("org.acme.Bar"))
+        .doTransform(t -> t.add(MyInterceptorBinding.class)));
+}
+----
```

Comment:
I think we should probably advertise the builder first, and only show that directly implementing the interface is possible after showing the builder. I confess that when writing https://github.com/Ladicek/cdi-extensions-demo, I didn't even realize we have a builder for annotation transformers.